### PR TITLE
Expose maxminddb metadata as a public method on Reader

### PR DESCRIPTION
**Pull request by andrewlook**
Carrying this over from @GregBowyer's pull request here: [ https://github.com/oschwald/geoip2-golang/pull/9 ], I wanted to make the changes that @oschwald recommended. Sorry if it's bad etiquette to make this a separate pullreq, but @GregBowyer and I were working on this together and he didn't give me access to his repo before he left the office (oops!)

@oschwald, I'm more than happy to squash these commits together if that would help keep your git history a little cleaner.

Also, when you asked for some documentation in @GregBowyer's PR, i'm assuming you meant as a function docstring and not as example code in the README?

### CODE REVIEW

File: reader.go [hunk 1]
```diff
@@ -111,6 +111,7 @@ type ISP struct {
 // using the Open function.
 type Reader struct {
 	mmdbReader *maxminddb.Reader
+	metadata   *maxminddb.Metadata
```
Comment (oschwald) [inline reader.go]: We don't need to store the metadata struct here anymore. `Metadata()` can access it from `mmdbReader` directly.

File: reader.go [hunk 2]
```diff
@@ -118,14 +119,22 @@ type Reader struct {
...
 // FromBytes takes a byte slice corresponding to a GeoIP2/GeoLite2 database
 // file and returns a Reader structure or an error.
 func FromBytes(bytes []byte) (*Reader, error) {
 	reader, err := maxminddb.FromBytes(bytes)
-	return &Reader{mmdbReader: reader}, err
+	if err == nil {
+		return &Reader{mmdbReader: reader, metadata: &reader.Metadata}, err
+	} else {
+		return &Reader{mmdbReader: reader}, err
+	}
```
Comment (oschwald) [inline reader.go]: This change and the one above can be reverted.

**Comment by oschwald** [comment]
Thanks, the API and docs looks like what I had in mind. I had one comment about the redundant storage of the metadata pointer.

**Comment by andrewlook** [comment]
@oschwald thanks for the feedback, should be good to go now!

**Comment by oschwald** [comment]
Thanks!

**Pull request merged**